src/commands/deployWorkspaceProject/deployWorkspaceProject.ts

```diff
@@ -186,24 +206,53 @@ export async function deployWorkspaceProject(context: IActionContext, item?: Con
     promptSteps.push(new ShouldSaveDeploySettingsPromptStep());
 
     const wizard: AzureWizard<DeployWorkspaceProjectContext> = new AzureWizard(wizardContext, {
-        title: localize('deployWorkspaceProjectTitle', 'Deploy workspace project to a container app'),
+        title: context.invokedFromApi ?
```

Comment:
We don't know the name of the client extension calling in, so just don't show a title when the API is triggered